app/assets/javascripts/page.js

```diff
@@ -0,0 +1,159 @@
+"use strict";
+
+() => {
+  var slugChecker = Backbone.Model.extend({
+      url: '/action_kit/check_slug',
+
+      defaults: {
+        valid: null,
+        slug: ''
+      }
+  });
+
+  var slugView = Backbone.View.extend({
+    el: '#new_page',
+
+    initialize() {
+      this.slugChecker = new slugChecker();
+      this.slugChecker.on("change:valid", _.bind(this.updateViewWithValid, this));
+      this.$title = this.$el.find('#page_title');
+      this.$slug  = this.$el.find('#page_slug');
+      this.$feedback = this.$el.find('.form-group.slug');
+      this.$checkButton = this.$el.find('#check_slug_available');
+      this.checking = false;
+    },
+
+    updateViewWithValid() {
+      var valid = this.slugChecker.get('valid');
+
+      this.$el.find('.form-group.slug').
```

Comment:
Backbone has the shortcut `this.$('.form-group.slug')` which does the same thing as `this.$el.find('.form-group.slug')` hahah you're the ideal use cause